app/components/episode-list-item.js

```diff
@@ -24,8 +25,9 @@ export default Ember.Component.extend({
           url: this.generateEpisodeURL(),
           description: this.shareTitle().replace("#", "%23"),
           ui: {
-            flyout: "top left",
-            button_text: ""
+            // flyout: "top left",
+            button_text: "Share",
```

Comment:
Identifier 'button_text' is not in camel case.<br>Mixed double and single quotes.